Azure/sonic-buildimage
# docker-sflow on buster

**- Why I did it**
This is an enhancement

**- How I did it**
Code change.

**- How to verify it**
Build `target/docker-sflow.gz`
Verify that `apt` refers to `buster` in `docker exec sflow cat /etc/apt/sources.list`

**- Description for the changelog**
Convert docker-sflow to buster


**- A picture of a cute animal (not mandatory but encouraged)**
